tools/aqrdev

```diff
@@ -256,12 +282,24 @@ def cmd_create(
         except BuildsPathNotFoundError:
             click.secho("Unable to find builds path", fg="red")
             sys.exit(errno.ENOENT)
+        for image_name, provider_name in avail_boxes:
+            if image_name == image:
+                box = image
+                provider = provider_name
+        click.secho(f"Imported image: '{box}' as Vagrant box Provider: {provider}", fg="green")
     elif box and image:
         click.secho("Please provide only one of '--box' or '--image'", fg="red")
         sys.exit(errno.EINVAL)
```

Comment:
I think for readability this should block should be checked first.

Reply:
@kshtsk no it was not there before and then added after. So it is really not my idea to change any logic but to add the provider support. Could you please don't review the logic because I didn't touch any of it.